AI/VCAI/MapObjectsEvaluator.h

```diff
@@ -0,0 +1,48 @@
+/*
+* MapObjectsEvaluator.h, part of VCMI engine
+*
+* Authors: listed in file AUTHORS in main folder
+*
+* License: GNU General Public License v2.0 or later
+* Full text of license available in license.txt file, in main folder
+*
+*/
+#pragma once
+
+struct AiMapObjectID
```

Comment:
This struct would be useful globally as part of CObjectClassesHandler

Reply:
What to do with it? Move to CObjectClassesHandler.h and what then?

Reply:
new getHandlerFor overload should accept it